mmcls/models/utils/augment/resizemix.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import torch
+from mmcv.utils import digit_version
+from torchvision.transforms import Resize
+
+from mmcls.models.utils.augment.builder import AUGMENT
+from .cutmix import BatchCutMixLayer
+from .utils import one_hot_encoding
+
+
+@AUGMENT.register_module(name='BatchResizeMix')
+class BatchResizeMixLayer(BatchCutMixLayer):
+    """ResizeMix Random Paste layer for batch ResizeMix.
+    Parameters
+    ----------
+    paste_min:
+        minimum value of lam
+    lam_max:
+        maximum value of lam
+    """
+
+    def __init__(self,
+                 lam_min: float = 0.1,
+                 lam_max: float = 0.8,
+                 *args,
+                 **kwargs):
+        if digit_version(torch.__version__) < digit_version('1.7.0'):
+            raise RuntimeError('torchvision.transforms.Resize is not available'
+                               'with Tensor before 1.7.0')
+        super(BatchResizeMixLayer, self).__init__(*args, **kwargs)
```

Comment:
It's better to list all arguments so that users can clearly find what they need pass.
```suggestion
    def __init__(self,
                 alpha,
                 num_classes,
                 lam_min: float = 0.1,
                 lam_max: float = 0.8,
                 prob=1.0,
                 cutmix_minmax=None,
                 correct_lam=True,
                 **kwargs):
        if digit_version(torch.__version__) < digit_version('1.7.0'):
            raise RuntimeError('torchvision.transforms.Resize is not available'
                               'with Tensor before 1.7.0')
        super(BatchResizeMixLayer, self).__init__(
            alpha=alpha,
            num_classes=num_classes,
            prob=prob,
            cutmix_minmax=cutmix_minmax,
            correct_lam=correct_lam,
            **kwargs)
```

Reply:
Consider adding an option `interpolation` to specify the mode of interpolation.